src/CLI/actioner/sonic-cli-vrf.py

```diff
@@ -45,13 +45,48 @@ def invoke_api(func, args=[]):
         keypath = cc.Path('/restconf/data/openconfig-network-instance:network-instances/network-instance={name}', name=args[0])
         return api.delete(keypath)
```

Comment:
sonic(config)# ip vrf vrf_red
%Error: Invalid VRF name

We should not throw an error when the VRF does not exist.

Reply:
Removed that error message.